terraform/modules/aws-network-load-balancer/resources.lb.tf

```diff
@@ -0,0 +1,68 @@
+resource "aws_lb" "nlb" {
+  name = "${var.environment}-loadbalancer"
+
+  internal           = false
+  load_balancer_type = "network"
+  enable_cross_zone_load_balancing = true
+
+  subnets = data.aws_subnet_ids.public_subnets.ids
+
+  tags = {
+    Environment = var.environment
+  }
+}
+
+
+resource "aws_lb_listener" "ingress" {
+  load_balancer_arn = aws_lb.nlb.arn
+
+  for_each = local.port_mapping
+
+  port     = lookup(each.value, "port")
+  protocol = lookup(each.value, "protocol")
```

Comment:
This would be way less complicated if made explicit.

Reply:
I can't follow. [Isn't it already explicit](https://github.com/wireapp/wire-server-deploy/pull/299/files#diff-02bca786de3aec768885210b90888324R1-R14)?